packages/web-components/src/slider/slider.styles.ts

```diff
@@ -0,0 +1,204 @@
+    --thumb-padding: 3px;
+  }
+  :host([orientation='horizontal']) {
+    touch-action: pan-y;
+    min-width: calc(var(--thumb-size) * 1px);
+    width: 100%;
+  }
+
+  :host([orientation='vertical']) {
+    --slider-direction: 0deg;
+    height: 160px;
+    min-height: var(--thumb-size);
+    touch-action: pan-x;
+    padding: 8px 0;
+  }
+
+  :host([disabled]:hover) {
+    cursor: initial;
+  }
+
+  :host(:focus-visible) {
+    outline-offset: 4px;
+    outline: 2px solid ${colorStrokeFocus2};
+  }
+
+  .thumb-cursor:focus {
+    outline: 0;
+  }
+
+  /* Thumb Container and Cursor */
+  .thumb-container {
+    position: absolute;
+    height: var(--thumb-size);
+    width: var(--thumb-size);
+    transition: all 0.2s ease;
+  }
+
+  :host([orientation='horizontal']) .thumb-container {
+    transform: translateX(calc(var(--thumb-size) * 0.5)) translateY(calc(var(--thumb-translate) * -1.5));
+  }
+
+  :host([orientation='horizontal'][size='small']) .thumb-container {
+    transform: translateX(calc(var(--thumb-size) * 0.5)) translateY(calc(var(--thumb-translate) * -1.35));
+  }
+
+  :host([orientation='vertical']) .thumb-container {
+    transform: translateX(calc(var(--thumb-translate) * -1.5)) translateY(calc(var(--thumb-size) * -0.5));
+  }
+
+  :host([orientation='vertical'][size='small']) .thumb-container {
+    transform: translateX(calc(var(--thumb-translate) * -1.35)) translateY(calc(var(--thumb-size) * -0.5));
+  }
+
+  .thumb-cursor {
+    height: var(--thumb-size);
+    width: var(--thumb-size);
+    background-color: ${colorBrandBackground};
+    border-radius: ${borderRadiusCircular};
+    box-shadow: inset 0 0 0 var(--thumb-padding) ${colorNeutralBackground1}, 0 0 0 1px ${colorNeutralStroke1};
+  }
+  .thumb-cursor:hover {
+    background-color: ${colorCompoundBrandBackgroundHover};
+  }
+  .thumb-cursor:active {
+    background-color: ${colorCompoundBrandBackgroundPressed};
+  }
+  :host([disabled]) .thumb-cursor {
+    background-color: ${colorNeutralForegroundDisabled};
+    box-shadow: inset 0 0 0 var(--thumb-padding) ${colorNeutralBackground1}, 0 0 0 1px ${colorNeutralStrokeDisabled};
+  }
+
+  /* Positioning Region */
+  .positioning-region {
+    position: relative;
+    display: grid;
+  }
+  :host([orientation='horizontal']) .positioning-region {
+    margin: 0 8px;
+    grid-template-rows: var(--thumb-size) var(--thumb-size);
+  }
+  :host([orientation='vertical']) .positioning-region {
+    margin: 8px 0;
+    height: 100%;
+    grid-template-columns: var(--thumb-size) var(--thumb-size);
+  }
+
+  /* Track */
+  .track {
+    align-self: start;
+    position: absolute;
+    background-color: ${colorNeutralStrokeAccessible};
+    border-radius: ${borderRadiusMedium};
+  }
+
+  :host([step]) .track::after {
+    content: '';
+    display: block;
+    position: absolute;
+    border-radius: ${borderRadiusMedium};
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    width: 100%;
+    inset: 0 -2px;
```

Comment:
inset is the shorthand for top, left, right, and bottom. both are not required. https://developer.mozilla.org/en-US/docs/web/css/inset